<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="RICHA" failures="1" tests="2" name="Error Validation Test" time="6.4" errors="0" timestamp="2024-04-24T23:49:23 IST">
  <testcase classname="rahulshettyacademy.tests.ErrorValidationsTest" name="loginErrorValidations" time="0.554">
    <skipped/>
  </testcase> <!-- loginErrorValidations -->
  <testcase classname="rahulshettyacademy.tests.ErrorValidationsTest" name="loginErrorValidations" time="1.085">
    <failure type="java.lang.AssertionError" message="expected [Login Successfully] but found [Incorrect email or password.]">
      <![CDATA[java.lang.AssertionError: expected [Login Successfully] but found [Incorrect email or password.]
at rahulshettyacademy.tests.ErrorValidationsTest.loginErrorValidations(ErrorValidationsTest.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- loginErrorValidations -->
  <testcase classname="rahulshettyacademy.tests.ErrorValidationsTest" name="productErrorValidations" time="0.0">
    <ignored/>
  </testcase> <!-- productErrorValidations -->
</testsuite> <!-- Error Validation Test -->
